File: Iniciante/1049 - Animal/Python/1049.py
def main():
    a = input()
    b = input()
    c = input()

    print(animal(a, b, c))


def animal(a, b, c):
    if a == "vertebrado" and b == "ave" and c == "carnivoro":
        return "aguia"
    elif a == "vertebrado" and b == "ave" and c == "onivoro":
        return "pomba"
    elif a == "vertebrado" and b == "mamifero" and c == "onivoro":
        return "homem"
    elif a == "vertebrado" and b == "mamifero" and c == "herbivoro":
        return "vaca"
    elif a == "invertebrado" and b == "inseto" and c == "hematofago":
        return "pulga"
    elif a == "invertebrado" and b == "inseto" and c == "herbivoro":
        return "lagarta"
    elif a == "invertebrado" and b == "anelideo" and c == "hematofago":
        return "sanguessuga"
    elif a == "invertebrado" and b == "anelideo" and c == "onivoro":
        return "minhoca"


main()

File: Iniciante/1049 - Animal/JavaScript/1049.js
function main() {
  var input = require("fs").readFileSync("/dev/stdin", "utf8");
  var lines = input.split("\n");

  const a = lines[0];
  const b = lines[1];
  const c = lines[2];

  console.log(animal(a, b, c));
}

function animal(a, b, c) {
  if (a == "vertebrado" && b == "ave" && c == "carnivoro") {
    return "aguia";
  } else if (a == "vertebrado" && b == "ave" && c == "onivoro") {
    return "pomba";
  } else if (a == "vertebrado" && b == "mamifero" && c == "onivoro") {
    return "homem";
  } else if (a == "vertebrado" && b == "mamifero" && c == "herbivoro") {
    return "vaca";
  } else if (a == "invertebrado" && b == "inseto" && c == "hematofago") {
    return "pulga";
  } else if (a == "invertebrado" && b == "inseto" && c == "herbivoro") {
    return "lagarta";
  } else if (a == "invertebrado" && b == "anelideo" && c == "hematofago") {
    return "sanguessuga";
  } else if (a == "invertebrado" && b == "anelideo" && c == "onivoro") {
    return "minhoca";
  }
}

main();
